packages/sanity/src/core/field/diff/components/ChangeBreadcrumb.tsx

```diff
@@ -1,17 +1,24 @@
-import React from 'react'
+import React, {useMemo} from 'react'
 import {Breadcrumbs, Text} from '@sanity/ui'
 import {ChangeTitlePath, FieldChangeNode} from '../../types'
 import {ChangeTitleSegment} from './ChangeTitleSegment'
 
 /** @internal */
 export function ChangeBreadcrumb(props: {change?: FieldChangeNode; titlePath: ChangeTitlePath}) {
   const {change, titlePath} = props
+  const maxLength = useMemo(() => {
+    if (change && change?.titlePath.length > 3) {
+      return 4
+    }
+    return undefined
+  }, [change])
 
   return (
     <Breadcrumbs
+      maxLength={maxLength}
       separator={
         <Text muted size={1}>
-          /
+          {'>'}
```

Comment:
**suggestion:** Use `<ChevronRightIcon />` (from `@sanity/icons`) instead of a greater than symbol

Reply:
Why do we even have that `maxWidth` set in any condition? Looks better without it? 

![Screenshot 2023-09-04 at 12 45 40](https://github.com/sanity-io/sanity/assets/44635000/b71ca9b4-c8e8-4955-a920-42efa499160a)

![Screenshot 2023-09-04 at 12 45 33](https://github.com/sanity-io/sanity/assets/44635000/0127604f-d54e-4c28-aeed-e93671c696c2)

Reply:
`maxWidth` probably made more sense before this `<Breadcrumbs>` component used `maxLength`. I agree in that it's probably no longer necessary.